samples/boards/sensortile_box_pro/ble-fw-upgrade/README.rst

```diff
@@ -0,0 +1,97 @@
+.. _sensortile_box_pro_sample_ble_fwupg:
+
+ST SensorTile.box Pro BLE Firwmare Upgrade
+##########################################
+
+Overview
+********
+This sample provides a simple method to upgrade the firmware for
+the BLE chip present on :ref:`sensortile_box_pro_board` board.
+
+It uses the BlueNRG-LPS UART bootloader protocol explained in `AN5471`_ .
+
+Requirements
+************
+
+The application requires a SensorTile.box Pro board connected to the PC
+through USB.
+
+To run this sample the console is not strictly needed, but might be useful.
+Following is an example for minicom.
+
+.. code-block:: console
+
+   $ minicom -D <tty_device> -b 115200
+
+Replace :code:`<tty_device>` with the correct device path automatically created on
+the host after the SensorTile.box Pro board gets connected to it,
+usually :code:`/dev/ttyUSBx` or :code:`/dev/ttyACMx` (with x being 0, 1, 2, ...).
```

Comment:
random sphinx trivia, you may make it look nicer using the below syntax:

```suggestion
usually :samp:`/dev/ttyUSB{x}` or :samp:`/dev/ttyACM{x}` (with x being 0, 1, 2, ...).
```

https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-samp

Reply:
ack

Reply:
@kartben Can you possibly take a look again after all the changes? Thanks